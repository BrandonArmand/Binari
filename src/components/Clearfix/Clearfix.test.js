import React from "react";
import { render } from "@testing-library/react";
import Clearfix from "./Clearfix";

describe("Clearfix", () => {
    it("renders snapshot", () => {
        const { container } = render(<Clearfix />);
        expect(container).toMatchSnapshot();
    });
});
