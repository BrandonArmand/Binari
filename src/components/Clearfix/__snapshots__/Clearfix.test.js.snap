// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clearfix renders snapshot 1`] = `
<div>
  <div
    class="makeStyles-clearfix-1"
  />
</div>
`;
